pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -64,31 +73,10 @@ var _ tektonpipelinereconciler.Finalizer = (*Reconciler)(nil)
 func (r *Reconciler) FinalizeKind(ctx context.Context, original *v1alpha1.TektonPipeline) pkgreconciler.Event {
 	logger := logging.FromContext(ctx)
 
-	// List all TektonPipelines to determine if cluster-scoped resources should be deleted.
-	tps, err := r.operatorClientSet.OperatorV1alpha1().TektonPipelines().List(ctx, metav1.ListOptions{})
-	if err != nil {
-		return fmt.Errorf("failed to list all TektonPipelines: %w", err)
-	}
-
-	for _, tp := range tps.Items {
-		if tp.GetDeletionTimestamp().IsZero() {
-			// Not deleting all TektonPipelines. Nothing to do here.
-			return nil
-		}
-	}
-
 	if err := r.extension.Finalize(ctx, original); err != nil {
 		logger.Error("Failed to finalize platform resources", err)
 	}
-	logger.Info("Deleting cluster-scoped resources")
-	manifest, err := r.installed(ctx, original)
-	if err != nil {
-		logger.Error("Unable to fetch installed manifest; no cluster-scoped resources will be finalized", err)
-		return nil
-	}
-	if err := common.Uninstall(ctx, manifest, nil); err != nil {
-		logger.Error("Failed to finalize platform resources", err)
-	}
+
```

Comment:
why does this finalizer don't have a Delete call on the associated InstallerSet.

Reply:
We can have a delete call, but as we have owner ref for InstallerSet, it will be garbage collected.
It will take some time to delete all resources after deleting TektonPipeline.
Deleting TektonPipeline would be when user want to uninstall and delete everything, so allowing it sometime to be garbage collected would be fine I think.
In case of InstallerSet controller might have to create a new so we delete all resources explicitly
wdyt?

Reply:
let us add the explict deletion to be on the safe side. 
Past experience has shown that we cannot rely 100% on gargbage colelction, as it is non-blocking.

Reply:
in addition, as we had the explict deletion before refactoring, let us add the explicit deletion. so that we don't get any surprises.
ref: https://github.com/tektoncd/operator/blob/dcf6156122cd8389673623640b3433d94b83d208/pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go#L89